tests/e2e/cucumber/environment/index.ts

```diff
@@ -15,8 +15,10 @@ import { api, store } from '../../support'
 import { World } from './world'
 import { state } from './shared'
 import { Browser, chromium, firefox, webkit } from 'playwright'
+import { camelCase } from 'lodash'
```

Comment:
```suggestion
import camelCase from 'lodash-es/camelCase'
```
Minor nitpick, but please lodash-es

Reply:
Ok, I change it now

Reply:
`import camelCase from 'lodash-es/camelCase'`  doesn't work for me. 
```
Error [ERR_REQUIRE_ESM]: require() of ES Module /Users/scharfviktor/Work/web/node_modules/lodash-es/camelCase.js from /Users/scharfviktor/Work/web/tests/e2e/cucumber/environment/index.ts not supported.
Instead change the require of camelCase.js in /Users/scharfviktor/Work/web/tests/e2e/cucumber/environment/index.ts to a dynamic import() which is available in all CommonJS modules.
```

Reply:
Damn, nevermind then ;) just keep it as-is